client/components/partySize.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+class PartySize extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			partySize: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20],
```

Comment:
Is there a specific reason why this partySize array is in state? It seems like your code is just mapping over the static array to generate options in a Select HTML element.